test/OpenTelemetry.Instrumentation.EventCounters.Tests/EventCountersMetricsTests.cs

```diff
@@ -267,15 +235,20 @@ private static double GetActualValue(Metric metric)
         return sum;
     }
 
-    private static bool AwaitExport<T>(MeterProvider meterProvider, List<T> exportedItems)
+    private static Metric AwaitExport(MeterProvider meterProvider, List<Metric> exportedItems, string expectedInstrumentName)
     {
-        return SpinWait.SpinUntil(
+        Metric metric = null;
+
+        SpinWait.SpinUntil(
             () =>
             {
                 Thread.Sleep(100);
                 meterProvider.ForceFlush();
-                return exportedItems.Count == 1;
+                metric = exportedItems.Where(x => x.Name == expectedInstrumentName).FirstOrDefault();
+                return metric != null;
```

Comment:
nit: This can be simplified as `return exportedItems.Any(x => x.Name == expectedInstrumentName)`